tests/resources/Group23-VIC-Machine-Service-Util.robot

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
```

Comment:
Can we rename this file to something more general, may be like `VIC-Machine-Service-Util.robot`? We may use it in other test suites as well.

Reply:
I don't want to give the file a more general name _because_ I don't want these keywords used in other test groups.

Our current process allows for only two types of keywords: suite-scoped and global. This file follows a different pattern: group-scoped.

Defining all shared keywords in the global scope carries many of the same drawbacks as defining variables in the global scope: identifying relevant keywords becomes harder, determining where keywords are used becomes harder, safely changing the behavior of keywords becomes harder, concisely and accurately naming keywords becomes harder, etc.

We'll iterate on this design as we learn more about this pattern, but I think this is a reasonable starting point.